script.js

```diff
@@ -0,0 +1,13 @@
+function shifumi() {
+
+    const user_choice = ['rock', 'paper', 'scissors']
+    let choice = prompt('Choose one of the following :' + user_choice[0] + ', ' + user_choice[1] + ', ' + user_choice[2])
+
+    if (user_choice.indexOf(choice) < 0) {
```

Comment:
It's working properly, but for several years now, we have another method in Javascript to do that directly. 
Instead of looping over the elements to find the index, there is a method checking directly if the element exists in an Array. I let you Google it :)

Reply:
found it I guess and easier !